components/CartPopupModal/CartPopupModal.js

```diff
@@ -151,6 +246,67 @@ const CartPopupModal = ({ type, item, selector, setCartState, disabled }) => {
         {type}
       </Button>
 
+      <Dialog
+        open={openConfirmation}
+        onClose={handleCloseConfirmation}
+        aria-labelledby='responsive-dialog-title'
+      >
+        <DialogTitle id='responsive-dialog-title'>
+          {'Proceed to deposit item?'}
```

Comment:
```suggestion
          Proceed to deposit item?
```

do we need to put it in `{}`? i think just writing it is ok